### ISSUE TITLE
invalid byte sequence in UTF-8

### ISSUE THREAD

**Issue by manuzhang** [dkhamsing/awesome_bot#91]
My file has some invalid sequences. Can we skip them or report better ?
```
/Users/doria/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/uri/rfc2396_parser.rb:267:in `scan': invalid byte sequence in UTF-8 (ArgumentError)
	from /Users/doria/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/uri/rfc2396_parser.rb:267:in `extract'
	from /Users/doria/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/uri/common.rb:293:in `extract'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/links.rb:32:in `links_find'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/check.rb:22:in `check'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/cli.rb:105:in `cli_process'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/cli.rb:47:in `block in cli'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/cli.rb:46:in `each'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/lib/awesome_bot/cli.rb:46:in `cli'
	from /Users/doria/.rvm/gems/ruby-2.2.1/gems/awesome_bot-1.12.0/bin/awesome_bot:6:in `<top (required)>'
	from /Users/doria/.rvm/gems/ruby-2.2.1/bin/awesome_bot:23:in `load'
	from /Users/doria/.rvm/gems/ruby-2.2.1/bin/awesome_bot:23:in `<main>'
	from /Users/doria/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/doria/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by dkhamsing** [comment]
Interesting bug, thanks for reporting
Will investigate

**Comment by dkhamsing** [comment]
Can you provide the sample file which caused this

**Comment by dkhamsing** [comment]
@manuzhang can you provide sample links?

**Comment by manuzhang** [comment]
https://github.com/gearpump/gearpump/blob/master/CHANGELOG.md

**Comment by dkhamsing** [comment]
Should have a fix soon, thanks for reporting this

**Comment by dkhamsing** [comment]
@manuzhang should be fixed in version `1.12.1` published today
